pkg/cloud/services/networking/floatingip.go

```diff
@@ -53,6 +56,13 @@ func (s *Service) GetOrCreateFloatingIP(openStackCluster *infrav1.OpenStackClust
 		return nil, err
 	}
 
+	tagOpts := attributestags.ReplaceAllOpts{Tags: []string{ip}}
```

Comment:
Should we add the floating ip as the tag or sth different? The issue is not explained precisely what we want to have as the tag.